scripts/common.sh

```diff
@@ -16,7 +16,7 @@ function clone_repository_and_move_inside {
   REPOSITORY_FOLDER=$(mktemp -d)
   echo "Created temporary directory ${REPOSITORY_FOLDER}"
 
-  git clone "https://${GITHUB_USERNAME}:${GITHUB_PERSONAL_ACCESS_TOKEN}@github.com/1024pix/${REPOSITORY_NAME}.git" "${REPOSITORY_FOLDER}"
+  git clone "https://${GITHUB_USERNAME}:${GITHUB_PERSONAL_ACCESS_TOKEN}@github.com/${ORGANIZATION_NAME}/${REPOSITORY_NAME}.git" "${REPOSITORY_FOLDER}"
```

Comment:
Suggestion nom de commit  `Extract organization name to variable (refacto)`
C'est un mélange de 
- ce que Fowler fait dans son bouquin (nommer pattern de refacto), voir le [catalogue](https://refactoring.com/catalog/?filter=tags-extract) de Fowler
- la [préco](https://blog.octo.com/en/seven-shades-of-git/#2) de LJA d'expliciter que c'est un refacto, et pas une fearure